test/e2e/cron/basic.yaml

```diff
@@ -3,7 +3,7 @@ kind: CronWorkflow
 metadata:
   name: test-cron-wf-basic
   labels:
-    argo-e2e: true
```

Comment:
Is there a reason for this change? Wondering why change type `bool` to `string`

Reply:
Especially since this is done in all test files

Reply:
this causes it to throw an error when `Unmarshal`ing to `unstructured.Unstructured` [here]( https://github.com/argoproj/argo-workflows/blob/b7f98e4859e912e8e8e279039c6d6e4f7fa5e209/workflow/common/parse.go#L42), Because labels are of type `map[string]string` and it doesn't like that `true` is bool. Not sure why it was allowed before, but there is a Kubernetes issue for it: https://github.com/kubernetes/kubernetes/issues/23251

Reply:
Huh, weird that it was allowed for so long

Reply:
yeah..